src/config.inc

```diff
@@ -8,6 +8,11 @@ else
   CXXFLAGS += -Wall -pedantic -Werror -Wno-deprecated-declarations -Wswitch-enum
 endif
 
+ifeq ($(CPROVER_WITH_PROFILING),1)
```

Comment:
I know you mentioned that somehow CMake unit tests fail if coverage is enabled, but I think it would still be preferable if we kept Make and CMake builds in sync, and at least adding these options to CMake would be good - even if you raise an issue in the GitHub issue tracker to cover whatever failures you see in CMake unit tests rather than also fixing them in this PR.

Reply:
True. The thing is this invariant violation happens only codebuild, and I've tried to reproduce it on local codebuild docker, it did not reproduce. I have a gut feeling though initially, cbmc-codecov environment was `openjdk-8`, not `standard-2.0`. Let me give me another go with cmake-coverage and if it's reproducible, I will raise an issue in the GitHub.

Reply:
@chrisr-diffblue , OK, so perhaps the invariant violation I observed was because of AWS image. I no longer observe it. I still have a problem codecov does not find existing coverage files from the VM. I will push just one more time with extra `LCOV` command it should be able to report. In that case, do you want another buildspec file ` buildspec-linux-cmake-gcc-cov.yml` and `CMakeLists.txt` to be part of this PR?